source/reference/older-version-support.txt

```diff
@@ -0,0 +1,93 @@
+.. _c2c-older-version-support:
+
+==================================
+Migrate Data From Pre-6.0 Clusters
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-pre-6.0-migration.rst
+
+About this Task 
+---------------
+
+``mongosync`` supports syncs between the following source and
+destination MongoDB server versions:
+
+.. include:: /includes/table-pre-6.0-compatibilty.rst
+
+Before You Begin 
+----------------
+
+:ref:`Download and install <c2c-install>` ``mongosync`` 1.10 or later.
+
+Embedded Verifier 
+~~~~~~~~~~~~~~~~~
+
+When you sync from a pre-6.0 source cluster, enable the 
+:ref:`embedded verifier <c2c-embedded-verifier>`. 
+
+Server Version Compatibility 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure that you are at least using a compatible patch version on your 
+source and destination clusters: 
+
+.. include:: /includes/table-patch-versions.rst
+
+To learn more about supported server versions, see 
+:ref:`c2c-server-version-compatibility`.
+
+Steps 
+-----
+
+In the following example, ``cluster0`` is an source cluster running MongoDB 6.0, 
```

Comment:
[nit]

This ought to be "a source cluster" or "the source cluster".

Reply:
ah typo that I left in from an earlier iteration. updating!